src/agent/LibFuzzerDotnetLoader/Program.cs

```diff
@@ -81,4 +70,107 @@ static bool TryTestOne<T>(MethodInfo method, Func<T, SharpFuzz.ReadOnlySpanActio
             return TryTestOne<TestOneArray>(method, t => span => t(span.ToArray()));
         }
     }
+
+    class LibFuzzerDotnetTarget
```

Comment:
This class became surprisingly verbose out of a desire to throw context-rich exceptions. Happy to accept C# tips on improving brevity. To some extent I'm just not used to the amount of vertical space eaten up by this brace style 😅.

Reply:
you can use a [record ](https://docs.microsoft.com/en-us/dotnet/csharp/whats-new/tutorials/records) instead if you want something smaller

Reply:
I think the class is more well-sized now, thanks to your other suggestions.